redwood-layout-uiview/src/commonMain/kotlin/app/cash/redwood/layout/uiview/UIViewFlexContainer.kt

```diff
@@ -135,7 +135,7 @@ internal class UIViewFlexContainer(
       setContentSize(
         CGSizeMake(
           width = container.items.maxOfOrNull { it.right } ?: 0.0,
-          height = container.items.maxOfOrNull { it.top } ?: 0.0,
+          height = container.items.maxOfOrNull { it.bottom } ?: 0.0,
```

Comment:
I don't know UIKit at all, but is `it.top` guaranteed to be 0 here? Do we need to do `bottom - top`?

Reply:
In UIKit the `top` of the bounds is 0 and `bottom` goes into the positive y axis, so the height is the largest bottom value.

Using `top` here caused the contentSize to be one view short, making items get cut off (more visible if you had a large item on the bottom because iOS adds a default content inset for the horizontal home indicator).